docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.md

```diff
@@ -13,23 +14,21 @@ This guide will help you instantiate a Data Context without a yml file, aka conf
 - See also our companion video for this guide: [Data Contexts In Code](https://youtu.be/4VMOYpjHNhM).
 :::
 
+## Steps
 
-Steps
------
+### 1. **Create a DataContextConfig**
 
-1. **Create a DataContextConfig**
+The `DataContextConfig` holds all of the associated configuration parameters to build a Data Context. There are defaults set for you to minimize configuration in typical cases, but please note that every parameter is configurable and all defaults are overridable. Also note that `DatasourceConfig` also has defaults which can be overridden.
 
-    The `DataContextConfig` holds all of the associated configuration parameters to build a Data Context. There are defaults set for you to minimize configuration in typical cases, but please note that every parameter is configurable and all defaults are overridable. Also note that `DatasourceConfig` also has defaults which can be overridden.
+Here we will show a few examples of common configurations, using the ``store_backend_defaults`` parameter. Note that you can use the existing API without defaults by omitting that parameter, and you can override all of the parameters as shown in the last example. A parameter set in ``DataContextConfig`` will override a parameter set in ``store_backend_defaults`` if both are used.
 
-    Here we will show a few examples of common configurations, using the ``store_backend_defaults`` parameter. Note that you can use the existing API without defaults by omitting that parameter, and you can override all of the parameters as shown in the last example. A parameter set in ``DataContextConfig`` will override a parameter set in ``store_backend_defaults`` if both are used.
+The following ``store_backend_defaults`` are currently available:
+    - :py:class:`~great_expectations.data_context.types.base.S3StoreBackendDefaults`
+    - :py:class:`~great_expectations.data_context.types.base.GCSStoreBackendDefaults`
+    - :py:class:`~great_expectations.data_context.types.base.DatabaseStoreBackendDefaults`
+    - :py:class:`~great_expectations.data_context.types.base.FilesystemStoreBackendDefaults`
```

Comment:
I think these are a holdover from the RST formatting. Could we clean this up and do something like:
```suggestion
The following ``store_backend_defaults`` are currently available in the `great_expectations.data_context.types.base` module:
    `S3StoreBackendDefaults`
    `GCSStoreBackendDefaults`
    `DatabaseStoreBackendDefaults`
    `FilesystemStoreBackendDefaults`
```

Reply:
We can certainly update how these are displayed, but I think it would be best to keep the bullet points as well.  Otherwise we may see some unusual formatting when this is rendered into html (markdown usually treats blocks of indented text as code blocks and disregards other markdown formatting in them).  I'll come back and make that edit/update after committing your other suggestions.

Reply:
Very good point - makes sense to me! I may have also left off a sufficient number of backticks.

Reply:
I've made and pushed this update.